docs/reference/mapping.asciidoc

```diff
@@ -84,26 +83,37 @@ If you increase this setting, we recommend you also increase the
 <<search-settings,`indices.query.bool.max_clause_count`>> setting, which
 limits the maximum number of <<query-dsl-bool-query,boolean clauses>> in a query.
 ====
++
+[TIP]
+====
+If your field mappings contain a large, arbitrary set of keys, consider using the <<flattened>> datatype.
```

Comment:
If you don't specify the anchor text here, "flattened" is capitalized: http://elasticsearch_56794.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/mapping.html#mapping-limit-settings
```suggestion
If your field mappings contain a large, arbitrary set of keys, consider using the <<flattened,flattened>> datatype.
```